db/fixtures/miq_product_features.yml

```diff
@@ -3359,6 +3359,11 @@
           :feature_type: control
           :hidden: true
           :identifier: delete_server
+        - :name: Delete Selected Servers
+          :description: Delete Selected Servers
+          :feature_type: control
+          :hidden: true
+          :identifier: servers_delete_server
```

Comment:
Is this a different feature from `:delete_server` above?

Reply:
No it isn't. In hindsight, this pr is unnecessary so I'll just close it and update the ui accordingly